### ISSUE TITLE
Find a way to support both HALs (`std` and `no_std`)

### ISSUE THREAD

**Issue by jessebraham** [esp-rs/esp-rust-board-bsp#4]
Currently this assumes that you will be using the `no_std` HAL, however it would be nice to support both. Maybe just add a `std` feature?

**Comment by romancardenas** [comment]
Good point. It would be also nice to be able to use the `direct-boot` feature of the `no_std` HAL, as well as being able to choose using RTT instead of JTAG.
